base/lppl.md

```diff
@@ -0,0 +1,197 @@
+
+4. If you are the Current Maintainer of the Work, you may, without restriction, modify the Work, thus creating a Derived Work. You may also distribute the Derived Work without restriction, including Compiled Works generated from the Derived Work. Derived Works distributed in this manner by the Current Maintainer are considered to be updated versions of the Work.
+
+5. If you are not the Current Maintainer of the Work, you may modify your copy of the Work, thus creating a Derived Work based on the Work, and compile this Derived Work, thus creating a Compiled Work based on the Derived Work.
+
+6. If you are not the Current Maintainer of the Work, you may distribute a Derived Work provided the following conditions are met for every component of the Work unless that component clearly states in the copyright notice that it is exempt from that condition. Only the Current Maintainer is allowed to add such statements of exemption to a component of the Work.
+
+     a. If a component of this Derived Work can be a direct replacement for a component of the Work when that component is used with the Base Interpreter, then, wherever this component of the Work identifies itself to the user when used interactively with that Base Interpreter, the replacement component of this Derived Work clearly and unambiguously identifies itself as a modified version of this component to the user when used interactively with that Base Interpreter.
+
+     b. Every component of the Derived Work contains prominent notices detailing the nature of the changes to that component, or a prominent reference to another file that is distributed as part of the Derived Work and that contains a complete and accurate log of the changes.
+
+     c. No information in the Derived Work implies that any persons, including (but not limited to) the authors of the original version of the Work, provide any support, including (but not limited to) the reporting and handling of errors, to recipients of the Derived Work unless those persons have stated explicitly that they do provide such support for the Derived Work.
+
+     d. You distribute at least one of the following with the Derived Work:
+
+          1. A complete, unmodified copy of the Work; if your distribution of a modified component is made by offering access to copy the modified component from a designated place, then offering equivalent access to copy the Work from the same or some similar place meets this condition, even though third parties are not compelled to copy the Work along with the modified component;
+
+          2. Information that is sufficient to obtain a complete, unmodified copy of the Work. 
```

Comment:
GitHub's markdown rendering (which as far as I know follows the markdown spec, so I think it can be trusted) understands these two items as blocks of code.  Removing four spaces before the `1.` and `2.` seems to fix the rendering (I didn't check the spec though).

Reply:
> An indented code block is composed of one or more indented chunks separated by blank lines. An indented chunk is a sequence of non-blank lines, each indented _four or more spaces_.

https://github.github.com/gfm/#indented-code-blocks

> Four spaces indent (of a list item) gives a code block:

https://github.github.com/gfm/#example-267

Reply:
crampdown has some specialties though. The above is not intended to be a code blockit is intended to be a sub-list within one item and you find that without the spaces it is not formatted as such but the numbering goes wrong.

on the other hand the numbering goes wrong anyway in places, which is why the thing is sitting there without being finished. I was getting more or less convinced that it is not a good idea to put the license into markdown unless it is actually faithfully representing the original (which I was up to now not able to achieve.

Reply:
I did a small research and it doesn't seem to be possible to get the rendered output _consistently_ correct.  With pure markdown I can see three options: 1) we trick markdown into believing that there is no list, so `1. whatever` is rendered in HTML as `<p>1. whatever</p>`, which kind of defeats the purpose of markdown but keeps the text faithful to the legal text; 2) keep the bottom-level list as a list, and do 1) for nested lists, so
```markdown
1. whatever
  a. nested
    1. nested nested
```
becomes in HTML
```html
<ol>
<li>
<p>whatever</p>
<p>a. nested</p>
<p>1. nested nested</p>
</li>
</ol>
```
which looks kinda better but very far from good; or 3) format everything as nested lists:
```markdown
1. whatever
   1. nested
      1. nested nested
```
which will keep the semantics in the rendered HTML:
```html
<ol><li>whatever
  <ol><li>nested
    <ol><li>nested nested</li></ol>
  </li></ol>
</li></ol>
```
but will rely on the implemented renderer to decide how to label each level (GitHub labels as 1., i., a.; stackexchange labels as 1., 1., 1.).

We could bite the bullet and write HTML directly into markdown, but even that seems to be ignored (GitHub) or not at all supported (stackexchange). This:
```html
<ol type="1"><li>whatever
  <ol type="a"><li>nested
    <ol type="1"><li>nested nested</li></ol>
  </li></ol>
</li></ol>
```
renders here, regardless of what I used in `type`, as:
<ol type="1"><li>whatever
  <ol type="a"><li>nested
    <ol type="1"><li>nested nested</li></ol>
  </li></ol>
</li></ol>

So for a text that needs cross-referencing like this one, markdown doesn't seem to be a good option.  You either have control on the meaning or on the appearance. Not both...  
Seems eerily similar to MS Word